sync/class.jetpack-sync-defaults.php

```diff
@@ -245,6 +263,19 @@ class Jetpack_Sync_Defaults {
 		'advanced_seo_description', // Jetpack_SEO_Posts::DESCRIPTION_META_KEY
 	);
 
+	public static function get_post_meta_whitelist() {
+		/**
+		 * Filter the list of post meta data that are manageable via the JSON API.
+		 *
+		 * @module sync
+		 *
+		 * @since 4.7
+		 *
+		 * @param array The default list of meta data keys.
+		 */
+		return apply_filters( 'jetpack_post_meta_whitelist', self::$post_meta_whitelist );
```

Comment:
Alright. But just to be clear, we will still have the pre-exsiting filter called `jetpack_options_whitelist`. Should I go ahead and also create `jetpack_sync_options_whitelist` ?